javagen/src/main/java/com/azure/autorest/mapper/ArrayMapper.java

```diff
@@ -9,29 +9,29 @@
 import java.util.Map;
 
 public class ArrayMapper implements IMapper<ArraySchema, IType> {
-    private static ArrayMapper instance = new ArrayMapper();
+    private static final ArrayMapper INSTANCE = new ArrayMapper();
     Map<ArraySchema, IType> parsed = new HashMap<>();
 
     private ArrayMapper() {
     }
 
     public static ArrayMapper getInstance() {
-        return instance;
+        return INSTANCE;
     }
 
     @Override
     public IType map(ArraySchema sequenceType) {
         if (sequenceType == null) {
             return null;
         }
-        if (parsed.containsKey(sequenceType)) {
-            return parsed.get(sequenceType);
-        }
-        IType iType = new ListType(Mappers.getSchemaMapper().map(sequenceType.getElementType()));
-        if (JavaSettings.getInstance().shouldUseIterable()) {
-            iType = new IterableType(Mappers.getSchemaMapper().map(sequenceType.getElementType()));
-        }
-        parsed.put(sequenceType, iType);
-        return iType;
+
+        return parsed.computeIfAbsent(sequenceType, sType -> {
```

Comment:
Seems having concurrency issue from CI.

Not sure what's different from vanilla and fluent. Only fluent CI runs on Java 11, while vanilla on Java 8.

```
error   | FluentGen | Failed to successfully run fluentgen plugin java.util.ConcurrentModificationException
java.util.ConcurrentModificationException
	at java.base/java.util.HashMap.computeIfAbsent(HashMap.java:1134)
	at com.azure.autorest.mapper.ArrayMapper.map(ArrayMapper.java:28)
	at com.azure.autorest.mapper.SchemaMapper.map(SchemaMapper.java:41)
	at com.azure.autorest.mapper.ModelMapper.map(ModelMapper.java:109)
	at com.azure.autorest.mapper.ClientMapper.lambda$map$6(ClientMapper.java:100)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.DistinctOps$1$2.accept(DistinctOps.java:175)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
	at com.azure.autorest.mapper.ClientMapper.map(ClientMapper.java:103)
	at com.azure.autorest.fluent.FluentGen.handleMap(FluentGen.java:175)
	at com.azure.autorest.fluent.FluentGen.processInternal(FluentGen.java:101)
	at com.azure.autorest.extension.base.plugin.NewPlugin.process(NewPlugin.java:206)
	at com.azure.autorest.fluent.Main.lambda$main$1(Main.java:18)
	at com.azure.autorest.extension.base.jsonrpc.Connection.lambda$dispatch$2(Connection.java:152)
	at com.azure.autorest.extension.base.jsonrpc.Connection.lambda$process$3(Connection.java:271)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```